script/openqa-clone-job

```diff
@@ -135,21 +145,38 @@ sub usage($) {
     }
 }
 
+sub split_jobid {
+    my ($url_string) = @_;
+    my $url          = Mojo::URL->new($url_string);
+    my $host_url     = $url->scheme . '://' . $url->host;
```

Comment:
I'd prefer using Mojo::URL here as well: `Mojo::URL->new->scheme($url->scheme)->host($url->host)->to_string`

Reply:
I will look into this in another PR crosschecking if there are more occurences than just this single line